azurerm/internal/services/cognitive/cognitive_account_resource.go

```diff
@@ -534,7 +534,7 @@ func resourceCognitiveAccountRead(d *pluginsdk.ResourceData, meta interface{}) e
 			return fmt.Errorf("setting `storages` for Cognitive Account %q: %+v", id, err)
 		}
 		if props.RestrictOutboundNetworkAccess != nil {
-			d.Set("outbound_network_access_restrited", *props.RestrictOutboundNetworkAccess)
+			d.Set("outbound_network_access_restrited", props.RestrictOutboundNetworkAccess)
```

Comment:
The name you set here looks a bit strange: "outbound_network_access_restrited" - should that last part not be "restricted" instead of "restrited"?